main/main.cpp

```diff
@@ -3500,6 +3506,11 @@ bool Main::start() {
 		}
 	}
 
+#ifdef TOOLS_ENABLED
+	if (editor) {
+		EDITOR_DEF_RST("input_devices/joypad/enabled", true);
```

Comment:
Yes, initially I was thinking about get this editor setting defined before the joypad is initialized, but it seems it's too early.

Reply:
The part that generates the documentation is tun on line 2905, hence why it needs to be moved up there 🙂

Reply:
Ah thanks, but anyway I think I need to move it to a later stage because now the EditorSettings is not instantiated yet. How stupid am I, I did not check the console error message. 

Now the problem is, the joypad is inited before the EditorSettings, so editor def and get will not work, which means under the current implentation, the joypad editor setting is useless ( User can still use a command line arg but that;'s more work ).

Reply:
You should probably register it in the editor settings file, but might not work in any case as you say